### PR TITLE
Updating copyright headers to match current best practices

### PR DESCRIPTION
**Pull request by tmichalak**
Added a short version of the ISC header to Python files

### CODE REVIEW

File: fasm/__init__.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
+#
```
Comment (mithro) [inline fasm/__init__.py:1]: Extra newline at the top of the file.

File: fasm/__init__.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
```
Comment (mithro) [inline fasm/__init__.py:2]: SymbiFlow Authors

File: fasm/__init__.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+#
```
Comment (mithro) [inline fasm/__init__.py:9]:
```suggestion

```

File: LICENSE [hunk 4]
```diff
@@ -1,4 +1,5 @@
-Copyright (C) 2012 - 2017  Clifford Wolf <clifford@clifford.at>
+Copyright (C) 2012-2017  Clifford Wolf <clifford@clifford.at>
```
Comment (tmichalak) [inline LICENSE:1]: @mithro @litghost this copyright was there from the beginning. Is it correct?

Comment (mithro) [reply LICENSE:1]: This does seem to be copy-pasta. The first commit in this repository was from @litghost in 2018 and https://github.com/SymbiFlow/fasm/graphs/contributors doesn't seem to list anything from @cliffordwolf?

@cliffordwolf - Do you recall contributing to this fasm stuff at all?

**Comment by daniellimws** [comment]
- [ ] Add or update the AUTHORS files to have the text as shown at https://opensource.google/docs/releasing/authors/ (Can get the names with `git shortlog -sn`)
- [ ] Add CI which checks for all this. (Can use these scripts https://github.com/SymbiFlow/fpga-tool-perf/tree/f8d143b73f670eea5efdbe96f1236d82998c896e/.github/travis)

**Comment by tmichalak** [comment]
@mithro If you are ok with the `AUTHORS` content then I think this can be merged.

**Pull request merged**